standard/ERCs/erc-6900.md

```diff
@@ -471,72 +470,55 @@ An account can have the same validation module installed more than once.
 The entityId of a validation function installed on an account MUST be unique.
 Validation installation MAY be deferred until a later time, such as upon first use.
 
-#### Responsibilties of `StandardExecutor` and `ModuleExecutor`
-
-`StandardExecutor` functions are used for open-ended calls to external addresses.
-
-`ModuleExecutor` functions are specifically used by modules to request the account to execute with account's context. Explicit permissions are required for modules to use `ModuleExecutor`.
-
-The following behavior MUST be followed:
-
-- `StandardExecutor` can NOT call module execution functions and/or `ModuleExecutor`. This is guaranteed by checking whether the call's target implements the `IModule` interface via ERC-165 as required.
-- `StandardExecutor` can NOT be called by module execution functions and/or `ModuleExecutor`.
-- Module execution functions MUST NOT request access to `StandardExecutor`, they MAY request access to `ModuleExecutor`.
-
-#### Calls to `installModule`
-
-The function `installModule` accepts 3 parameters: the address of the module to install, the Keccak-256 hash of the module's manifest, ABI-encoded data to pass to the module's `onInstall` callback.
-
-The function MUST retrieve the module's manifest by calling `moduleManifest()` using `staticcall`.
-
-The function MUST perform the following preliminary checks:
+During validation installation, the account MUST correctly set flags and other fields based on the incoming data provided by users.
 
-- Revert if the module has already been installed on the modular account.
-- Revert if the module does not implement ERC-165 or does not support the `IModule` interface.
-- Revert if `manifestHash` does not match the computed Keccak-256 hash of the module's returned manifest. This prevents installation of modules that attempt to install a different module configuration than the one that was approved by the client.
-- Revert if any address in `dependencies` does not support the interface at its matching index in the manifest's `dependencyInterfaceIds`, or if the two array lengths do not match, or if any of the dependencies are not already installed on the modular account.
+- the account MUST install all pre validation hooks required by the user and call `onInstall` with the user-provided data on the hook module to initialize the states.
+- the account MUST install all permission hooks required by the user and call `onInstall` with the user-provided data on the hook module to initialize the states.
+- the account MUST add all selectors required by the user that the validation can validate.
+- the account MUST set `isGlobal` and `isSignatureValidation` flags as required.
+- the account MUST call `onInstall` on the validation module to initialize the states.
```

Comment:
```suggestion
- the account MAY call `onInstall` on the validation module to initialize the states.
```

I like emphasizing that this is kind of a toss-up, so hooks should be built knowing they may not always be called with `onInstall` (this doesn't mean they can't require it, but they should be ready to handle cases where they weren't called and revert or just early-out or something).

Reply:
Maybe "SHOULD" instead?